tensorflow/tools/pip_package/build_pip_package.sh

```diff
@@ -89,6 +89,8 @@ function main() {
       bazel-bin/tensorflow/tools/pip_package/build_pip_package.runfiles/external \
       "${TMPDIR}/external"
     RUNFILES=bazel-bin/tensorflow/tools/pip_package/build_pip_package.runfiles
+		cp -R \
+			bazel-bin/_solib_k8 "${TMPDIR}"
```

Comment:
I don't know what this is doing -- this deserves a comment, and it's unclear why this is the right command to use -- should the path be more specific than just bazel-bin/_solib_k8 ?